daemon/lxd.go

```diff
@@ -166,3 +169,92 @@ func (l *LXD) DeleteContainer(hostname string) (bool, error) {
 
 	return true, nil
 }
+
+func (l *LXD) CreateContainerFile(container pfmodel.Container, fullPath string) error {
+	var content string
+	mode := 644
+	contentType := "file"
+	writeMode := "overwrite"
+
+	// // prepare filescript
+	for _, bs := range container.Bootstrappers {
+		switch bs.Type {
+		case "chef-solo":
+			content = `
```

Comment:
can content of the file be put as a file somewhere else and read on the fly here?

Reply:
since we need bootstrapper data. now, we move it to be different func. please check